sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/optimizer/InjectRuntimeFilter.scala

```diff
@@ -146,6 +148,15 @@ object InjectRuntimeFilter extends Rule[LogicalPlan] with PredicateHelper with J
           predicateReference ++ condition.references,
           hasHitFilter = true,
           hasHitSelectiveFilter = hasHitSelectiveFilter || isLikelySelective(condition))
+      case ExtractEquiJoinKeys(joinType, leftKeys, rightKeys, _, _, left, right, _)
```

Comment:
The PR description is very clear about what this PR is doing, but the "how" part is missing. I don't quite understand the changes here. Does it mean a join query can be the build query of a runtime filter?

Reply:
The selective is not only exists in brother node, it also exists in left or right of child join node.